scripts/index.js

```diff
@@ -0,0 +1,80 @@
+const cityName = document.querySelector('.city-name');
+const cityNameForm = document.querySelector('#city-name-form');
+const temp = document.querySelector('#temp');
+const plus = document.querySelector('#plus');
+const minus = document.querySelector('#minus');
+const landscape = document.querySelector('.landscape');
+const skyBackground = document.querySelector('.sky-display');
+const skyMenu = document.querySelector('.sky-menu');
+const skyMenuForm = document.querySelector('#sky-menu-form');
+
+cityNameForm.addEventListener('submit', (event) => {
+    let info = new FormData(cityNameForm)
+    for (var value of info.values()) {
+        cityName.textContent = value;
+    } 
+    cityNameForm.style.display = 'none';  
+    event.preventDefault();
+});
```

Comment:
instead of a `submit` event listener, maybe we can research `keyup` to catch each character that is typed and displayed on the html